### PR TITLE
Add parameter to example code

### LINKED ISSUE #13379: Misleading example code for challenge
#### Challenge Name

[Coding Interview Data Structure Questions: Create a Set Class](http://beta.freecodecamp.com/en/challenges/coding-interview-data-structure-questions/create-a-set-class)

#### Issue Description

For this challenge, the user is asked to write an add function for a set. The description gives a template function which takes in no parameters:

image

However, this isn't correct because the function needs to have one parameter, the element to add. This is also evident from the seed code in the [next challenge](http://beta.freecodecamp.com/en/challenges/coding-interview-data-structure-questions/remove-from-a-set):

image

### PR DESCRIPTION
**Pull request by jamos-tay**
#### Pre-Submission Checklist

- [X] Your pull request targets the `staging` branch of freeCodeCamp.
- [X] Branch starts with either `fix/`, `feature/`, or `translate/` (e.g. `fix/signin-issue`)
- [X] You have only one commit (if not, [squash](http://forum.freecodecamp.com/t/how-to-squash-multiple-commits-into-one-with-git/13231) them into one commit).
- [X] All new and existing tests pass the command `npm test`. Use `git commit --amend` to amend any fixes.

#### Type of Change

- [X] Small bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds new functionality)
- [ ] Breaking change (fix or feature that would change existing functionality)
- [ ] Add new translation (feature adding new translations)

#### Checklist:

- [X] Tested changes locally.
- [X] Closes currently open issue (replace XXXX with an issue no): Closes #13379

#### Description

Added `element` as parameter in example code.

Also converted block code to `blockquote` form, as it looks nicer

### CODE REVIEW

File: seed/challenges/08-coding-interview-questions-and-take-home-assignments/coding-interview-data-structure-questions.json [hunk 1]
```diff
@@ -227,15 +227,10 @@
       "title": "Create a Set Class",
       "description": [
         "In the next few exercises we are going to create a function to emulate a data structure called a \"Set\". A Set is like an array, but it cannot contain duplicate values. The typical use for a Set is to simply check for the presence of an item. This can be implemented with an object, for instance:",
-        "<code>var set = new Object();</code>",
-        "<code>set.foo = true;</code>",
-        "<code>// See if foo exists in our set:</code>",
-        "<code>console.log(set.foo) // true</code>",
+        "<blockquote>var set = new Object();<br>set.foo = true;<br>// See if foo exists in our set:<br>console.log(set.foo) // true</blockquote>",
         "In the next few exercises, we will build a full featured Set from scratch.",
         "In this exercise you will add a function to add a value to our set collection:",
```
Comment (dhcodes) [inline seed/challenges/08-coding-interview-questions-and-take-home-assignments/coding-interview-data-structure-questions.json]: While you're at it, can you change this line from:
>In this exercise you will add a function to add a value to our set collection:

to:

>For this exercise, create a function that will add a value to our set collection as long as the value does not already exist in the set. For example:

**Review by dhcodes — CHANGES_REQUESTED** [review]

**Comment by dhcodes** [comment]
@jamos-tay I think we could improve it further. Can you change the line in the review on your local branch and then run `git commit --amend` and save the same commit message, and then push your branch again with `-f` to update this PR.

Thanks.

**Comment by jamos-tay** [comment]
@dhcodes Okay, I've made the necessary changes 👍

**Review by dhcodes — APPROVED** [review]

**Comment by dhcodes** [comment]
Tested locally and **LGTM!**. Thanks for contributing to freeCodeCamp! 👍 🌮 🎉

**Pull request merged**